warehouse/models/mart/benefits/_mart_benefits.yml

```diff
@@ -57,10 +39,6 @@ models:
         description: string
       - name: region
         description: string
-      - name: dma
-        description: string
-      - name: group_properties
-        description: dict
       - name: event_properties
         description: dict
       - name: user_properties
```

Comment:
@allejo I think the JSON columns have to be removed from the model definition as well.

Reply:
Damn it! I knew I felt like I missed something